op-chain-ops/safe/batch.go

```diff
@@ -25,7 +25,7 @@ import (
 // the raw calldata when both the calldata and ABIs with arguments are
 // present.
 type Batch struct {
-	SkipCalldata bool
+	SkipCalldata bool               `json:"-"`
```

Comment:
nit: probably more idiomatic to make this a private field if it's not used outside this package.

Reply:
Then would need to make a `NewBatch(bool)` function to be able to set it